Resources/Private/Language/de.locallang.xlf

```diff
@@ -461,6 +469,71 @@
 				<source><![CDATA[Zoom Out]]></source>
 				<target><![CDATA[Ansicht verkleinern]]></target>
 			</trans-unit>
+            <trans-unit id="newTenant.formatOkay" approved="yes">
+                <source><![CDATA[Default namespaces found!]]></source>
+                <target><![CDATA[Standard-Namensräume gefunden!]]></target>
+            </trans-unit>
+            <trans-unit id="newTenant.formatOkayMsg" approved="yes">
+                <source><![CDATA[The namespace definitions for MODS, TEIHDR and ALTO do exist.]]></source>
+                <target><![CDATA[Die Standard-Namensräume für MODS, TEIHDR und ALTO sind definiert.]]></target>
+            </trans-unit>
+            <trans-unit id="newTenant.formatCreated" approved="yes">
+                <source><![CDATA[Default namespaces created successfully!]]></source>
+                <target><![CDATA[Standard-Namensräume erfolgreich angelegt!]]></target>
+            </trans-unit>
+            <trans-unit id="newTenant.formatCreatedMsg" approved="yes">
+                <source><![CDATA[The namespace definitions for MODS, TEIHDR and/or ALTO were created properly.]]></source>
+                <target><![CDATA[Die Standard-Namensräume für MODS, TEIHDR und/oder ALTO wurden erfolgreich definiert.]]></target>
+            </trans-unit>
+            <trans-unit id="newTenant.formatNotCreated" approved="yes">
+                <source><![CDATA[Default namespaces not found!]]></source>
+                <target><![CDATA[Standard-Namensräume nicht gefunden!]]></target>
+            </trans-unit>
+            <trans-unit id="newTenant.formatNotCreatedMsg" approved="yes">
+                <source><![CDATA[The namespace definitions for MODS, TEIHDR and/or ALTO do not exist and could not be created.]]></source>
+                <target><![CDATA[Die Standard-Namensräume für MODS, TEIHDR und/oder ALTO konnten nicht definiert werden.]]></target>
+            </trans-unit>
```

Comment:
These messages don't seem to be used.

Reply:
Correct. I removed all these flash messages. You land on the page with the New Tenant module and see, that nothing changed. I think, this is enough.